coderedcms/templates/coderedcms/blocks/accordion_block.html

```diff
@@ -0,0 +1,19 @@
+{% extends 'coderedcms/blocks/base_block.html' %}
+
+{% load wagtailcore_tags coderedcms_tags %}
+
+{% block block_render %}
+
+{% generate_random_id as modal_id %}
+<button
```

Comment:
Let's make this template match the bootstrap docs example as closely as possible. Generally speaking the bootstrap examples are a baseline for our components.

https://getbootstrap.com/docs/4.6/components/collapse/#accordion-example